src/Raven.Server/Smuggler/Documents/Processors/BuildVersion.cs

```diff
@@ -6,12 +6,16 @@ public static class BuildVersion
     {
         public static BuildVersionType Type(long buildVersion)
         {
-            if (ServerVersion.IsNightlyOrDev(buildVersion))
+            if (buildVersion >= 50 && buildVersion < 60)
+                return BuildVersionType.V5; // debug / dev version
+            if (buildVersion >= 40 && buildVersion < 50)
                 return BuildVersionType.V4; // debug / dev version
             if (buildVersion < 40000)
                 return BuildVersionType.V3;
             if (buildVersion >= 40000 && buildVersion <= 49999)
                 return BuildVersionType.V4;
+            if (buildVersion >= 50000 && buildVersion <= 59999)
+                return BuildVersionType.V5;
```

Comment:
What checks in Smuggler we are doing based on version?

Reply:
https://github.com/ravendb/ravendb/pull/10417/files#diff-f1e6d3944a0fbb464ee785bbc0f410d1R34

Reply:
https://github.com/ravendb/ravendb/blob/d0513b774ba3ac5735cb8191f80d9f66082b5e75/src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs#L662-L672

I think it would be better to introduce BuildVersionType.GreaterThanCurrent or sth similar